stdlib/universe/group.go

```diff
@@ -242,6 +279,55 @@ func (t *groupTransformation) appendTable(ab *table.BufferedBuilder, tbl flux.Ta
 	return ab.AppendTable(tbl)
 }
 
+func (t *groupTransformation) groupChunkByRow(tbl table.Chunk, d *execute.TransportDataset, mem arrowmem.Allocator) error {
```

Comment:
I'd like to see us consolidate this with the existing one, but nothing is coming to mind.